traverse_builtins/ppxlib_traverse_builtins.ml

```diff
@@ -12,6 +12,9 @@ class map =
   let any x = x in
   object
     method int : int T.map = any
+    method int32 : int32 T.map = any
```

Comment:
These seem like a worthy addition but I think I missed why we need them now, could you provide some context?

Reply:
Hmm I have no idea, I can actually remove them and everything still builds! Perhaps I was working on something else at the time and thought I needed them. Happy to remove them from this PR :) ?

Reply:
Yeah I guess if we don't use them we might as well remove them!